binding/ios/PorcupineManager.swift

```diff
@@ -0,0 +1,131 @@
+//
```

Comment:
Please fo block comments use the /* */ as described here: https://developer.apple.com/library/content/documentation/Xcode/Reference/xcode_markup_formatting_ref/ComentBlock.html

Reply:
I think that's for markup. xcode inserts // and looking at Foundation classes seems like that is the preferred method.